homeassistant/components/emoncms/sensor.py

```diff
@@ -107,6 +106,10 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         if sensor_names is not None:
             name = sensor_names.get(int(elem["id"]), None)
 
+        unit_of_measurement = elem.get("unit", "")
```

Comment:
`dict.get()` already returns None so we should work with that instead of specifying a default value as an empty string. Moreover, we should use a new variable.
```suggestion
        unit = elem.get("unit")
```